docs/blog/2020-02-17-changing-the-oss-status-quo/index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Changing the OSS Status Quo"
+date: 2020-02-17
+author: Aisha Blake
+excerpt: "How can we increase the diversity of open source contributors? As maintainers, we need to listen to a diverse group of community members and make thoughtful changes to create a truly inclusive project."
+tags:
+  - open-source
+  - community
+---
+
+In business and in our individual careers, so many of us rely heavily on open source software. We need it but open source isn't as open as it’s often made out to be. Too many projects are unfriendly, even openly hostile, to new contributors. This effect, combined with the intersection of many underrepresented identities in tech (BIPOC and women being some of the most visible), effectively shuts people out of the process. Consider who contributes to open source.
+
+The status quo is made and remade every time you as a maintainer add to your documentation. Every time you, as a maintainer, engage with the community via social media, issue queues, and your documentation, you help decide who contributes. You contribute to their safety and their productivity. Right now, barriers to contribution are severely limiting our capacity to create and sustain open source projects.
+
+There's an opportunity here. There are thousands of highly skilled potential contributors ready, willing, and able to contribute to open source projects but they need to know that they'll be safe in doing so. A handful of projects are intentionally focusing on this work and making incredible strides towards a more inclusive open source community.
+
+Gatsby's most admirable quality is the company’s commitment to that community. Yes, it's a great tool but what use is any tool that is too difficult to use or too obscure for most people to access? One of Gatsby's core values is “You Belong Here”. They back it up with a prominent, clear code of conduct that they consistently reference and enforce. The documentation is vast and includes specifics around contributing as well as providing materials that make it easy for  people outside of the core team to teach as well as learn. New contributors who have had a PR merged in are invited to the GitHub organization and offered SWAG as a thank you. There are even a limited number of one-on-one mentorship opportunities available.
```

Comment:
I feel like "we" makes more sense now that I've been hired. Also, so far, my only experience with what I saw as a code of conduct violation involved having to prod a bit to get some consensus and then eventually action. How do people feel we do with this in general?

Reply:
Issues related to the Code of Conduct come up so rarely that we haven't had to do much–I've seen it come up once on GitHub and a comment to the person sufficed (they're still contributing, and doing a fine job).